commands/transform.go

```diff
@@ -93,6 +93,19 @@ func transformSlackCmdF(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if !skipAttachments {
```

Comment:
We may want to activate this path only if the flag is set. (eg.  `--include-profile-pictures`)

Reply:
For consistency with the other flags I have introduced it as `--skip-profile-pictures`, with a default set to `false`.